packages/compartment-mapper/src/archive.js

```diff
@@ -16,7 +16,7 @@ import { writeZip } from '@endo/zip';
 import { resolve } from './node-module-specifier.js';
 import { compartmentMapForNodeModules } from './node-modules.js';
 import { search } from './search.js';
-import { assemble } from './assemble.js';
+import { link } from './link.js';
```

Comment:
Everything now imports `link` from `link.js`, but the rename from `assemble.js` to `link.js` entailed no other changes.  How does that work?  Don't you need to export `link` instead of `assemble`?

Reply:
Rather than rename `assemble` to `link`, I refactored `assemble` such that it's implemented in terms of `link`, but `link` returns a record containing the compartment instead of just the root compartment. This was an unrelated change needed to make it possible for `link` to return other things, like a hash. Assemble is still there, but deprecated.